presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -440,6 +440,14 @@ Create an Iceberg table partitioned by the first letter of the team field::
     ``Day``, ``Month``, ``Year``, ``Hour`` partition column transform functions are not supported in Presto Iceberg
     connector yet (:issue:`20570`).
 
+CREATE VIEW
+^^^^^^^^^^^^
+
+Currently Iceberg connector supports creating views in Hive and Glue metastores.
+Create a view named ``view_page_views`` on top of ``iceberg.web.page_views`` table created above as given below::
```

Comment:
```suggestion
To create a view named ``view_page_views`` for  the ``iceberg.web.page_views`` table created in the `CREATE TABLE`_ example::
```

Because documentation pieces can get moved around, "above" and "below" are fragile. Avoiding them when possible can future-proof the documentation from becoming out of date through rearranging. Added a link to the CREATE TABLE example to help the user find it - this is a long page and they could scroll past it looking for it.

Reply:
Changed.